main.js

```diff
@@ -1,33 +1,19 @@
 module.exports = {
   configs: {
-    parity: {
```

Comment:
I'd be so sad if `eslint` really couldn't override every rule it had, for programmability's sake... seems to make the "multi-config" thing pointless, where we could have multiple sets (including historical ones) prefixed with `bestow`. But if we really can't, then 😢 

Is it the `prettier` rules? I would be sad. One day, I really hoped to go write some code in here (thus the js instead of json) so we could have things like anonymous generators and spaces between curried params. But maybe we can just approve and try to fork `eslint` or `prettier` rules later.